blueprints/codes/cur/cur_228/formula_2_22.py

```diff
@@ -0,0 +1,68 @@
+"""Contains formula 2.22 from CUR 228."""
+
+from blueprints.codes.cur.cur_228 import CUR_228
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import KN_M3, KPA, M
+
+
+class Form2Dot22MenardStiffness(Formula):
+    """Representation of equation 2.22 CUR 228."""
+
+    source_document = CUR_228
+    label = "2.22"
+
+    def __init__(self, r: M, e_p: KPA, alpha: float) -> None:
+        """Calculates the Menard stiffness (k_h) for r < 0.3 m.
+
+        Parameters
+        ----------
+        r: M
+            The radius of a foundation pile [m]:
+            r >= 0.3 m
+        e_p: KPA
+            Elastic modulus of Ménard [kPa]:
+            e_p ≈ beta * q_c
+                beta: float
+                    Dependent on soil type [-]:
+                q_c: KPA
+                    Cone resistance [kPa]
+        alpha: float
+            Factor dependent on soil type [-]:
+        """
+        super().__init__()
+        self.r = r
+        self.e_p = e_p
+        self.alpha = alpha
+
+    @staticmethod
+    def _evaluate(r: M, e_p: KPA, alpha: float) -> KN_M3:
+        """Return the Menard stiffness k_h when r >= 0.3 m [kN/m3]."""
```

Comment:
add checks for negative numbers
```suggestion
        """Return the Menard stiffness k_h when r >= 0.3 m [kN/m3]."""
        raise_if_negative(r=r, e_p=e_p, alpha=alpha)
```

Reply:
Added